Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -1605,15 +1617,31 @@ namespace Babylon
             XRRigidTransform& m_transform;
             Napi::ObjectReference m_jsPose{};
             Napi::ObjectReference m_jsJointPose{};
+            
+            bool m_hasBegunTracking = false;
 
```

Comment:
nit:  i think the preference is initialization in brackets

Reply:
Updated